tool/jt.rb

```diff
@@ -708,42 +710,97 @@ def compile_cext(name, ext_dir, target, *clang_opts)
     end
   end
 
-  def pr(*args)
-    out, _err   = sh 'git', 'remote', capture: true
-    remotes     = out.split
-    remote_urls = remotes.map do |remote|
-      out, _err = sh 'git', 'config', '--get', "remote.#{remote}.url", capture: true
-      [remote, out.chomp!]
-    end
+  module PR
+    include ShellUtils
 
-    upstream = remote_urls.find { |r, u| u.include? 'graalvm/truffleruby' }.first
-    bb       = remote_urls.find { |r, u| u.include? 'ol-bitbucket' }.first
+    def pr_clear
+      require 'net/http'
+      require 'pp'
 
-    # Fetch PRs on GitHub
-    fetch = "+refs/pull/*/head:refs/remotes/#{upstream}/pr/*"
-    out, _err = sh 'git', 'config', '--get-all', "remote.#{upstream}.fetch", capture: true
-    sh 'git', 'config', '--add', "remote.#{upstream}.fetch", fetch unless out.include? fetch
-    sh 'git', 'fetch', upstream
+      bb  = bb(remote_urls)
+      uri = URI('https://api.github.com/repos/graalvm/truffleruby/pulls')
+      puts "Contacting GitHub: #{uri}"
+      data     = Net::HTTP.get(uri)
+      prs_data = JSON.parse data
+      open_prs = prs_data.map { |prd| prd.fetch 'number' }
```

Comment:
It would be better to wrap the value with `Integer(prd.fetch('number'))` here rather than a weak `to_i` later.